client/src/app/gui.rs

```diff
@@ -47,7 +44,8 @@ pub struct Gui {
     is_capturing: bool,
     // some kind of sparse vector would be the best
     // but this will do
-    packets: Packets,
+    packets: RefPackets,
+    streams: RefStreams,
```

Comment:
Can we get rid of packets: RefPackets and use ones that held in streams?

Reply:
We could do that, I was also thinking of it.